src/main/java/org/hypertrace/gradle/publishing/PublishPlugin.java

```diff
@@ -121,7 +122,10 @@ private void configureUploadTask(BintrayUploadTask task) {
     task.setPackageName(this.extension.name.get());
     task.setPackageVcsUrl(this.extension.vcsUrl.get());
     task.setUserOrg(this.extension.organization.get());
-    task.setPackageLicenses(this.extension.license.get().bintrayString);
+    if (this.extension.license.get() != TRACEABLE_COMMUNITY) {
+      // The bintray plugin doesn't support custom licenses. This is the default on all repos.
+      task.setPackageLicenses(this.extension.license.get().bintrayString);
```

Comment:
what does this do internally? I don't think it puts the license file so what else?

Reply:
the bintray plugin's api only supports the `licenses` field (OSS licenses) instead of the `custom_licenses` field that the rest api requires for custom ones. Instead, our repos are set to default the custom license always, so we treat this as a default case and leave it unset. For apache, we set the string. See https://bintray.com/docs/api/#_create_package